avm/res/service-fabric/cluster/tests/e2e/defaults/main.test.bicep

```diff
@@ -45,6 +45,10 @@ module testDeployment '../../../main.bicep' = [
       name: '${namePrefix}${serviceShort}001'
       managementEndpoint: 'https://${namePrefix}${serviceShort}001.westeurope.cloudapp.azure.com:19080'
       reliabilityLevel: 'None'
+      certificate: {
```

Comment:
Is this a required parameter? Asking because this is the default test

Reply:
There was an error stating deploying an unsecure cluster is not allowed. The test case wouldn't passw without using a certificate.

Reply:
If required, the corresponding description should be updated with parameter type Required. But only if that is the case